specs/IsSwipeNavigationEnabled.md

```diff
@@ -0,0 +1,93 @@
+
+In this document we describe the new setting. We'd appreciate your feedback.
+
+# Description
+The default value for `IsSwipeNavigationEnabled` is `TRUE`.
+When this setting is set to `FALSE`, it disables the ability of the end users to use swiping gesture on touch input enabled devices to trigger navigations such as swipe left/right to go back/forward or pull to refresh current page.
+Disabling/Enabling `IsSwipeNavigationEnabled` does not take effect until the next navigation.
+
+
+# Examples
+```cpp
+wil::com_ptr<ICoreWebView2> webView;
+void SettingsComponent::ToggleSwipeNavigationEnabled()
+{
+    // Get webView's current settings
+    wil::com_ptr<ICoreWebView2Settings> coreWebView2Settings;
+    CHECK_FAILURE(webView->get_Settings(&coreWebView2Settings));
+
+    wil::com_ptr<ICoreWebView2Settings6> coreWebView2Settings6;
+    coreWebView2Settings6 = coreWebView2Settings.try_query<ICoreWebView2Settings6>();
+    if(coreWebView2Settings6) 
+    {
+        BOOL enabled;
+        CHECK_FAILURE(coreWebView2Settings6->get_IsSwipeNavigationEnabled(&enabled));
+        CHECK_FAILURE(coreWebView2Settings6->put_IsSwipeNavigationEnabled(enabled ? FALSE : TRUE));
+    }
+}
+```
+
+```c#
+private WebView2 _webView;
+void ToggleSwipeNavigationEnabled()
+{
+    var coreWebView2Settings = _webView.CoreWebView2.Settings;
+    coreWebView2Settings.IsSwipeNavigationEnabled = !coreWebView2Settings.IsSwipeNavigationEnabled;
+}
+```
+
+# Remarks
+This API only affects the decision whether to trigger navigations of the main html page when end users perform overscrolling motions by swiping and has no effect on the scrolling interaction used to explore the web content shown in WebView2 or the CSS overscroll-behavior property.
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+```cpp
+[uuid(45b1f964-f703-47ac-a19a-b589dd0c5559), object, pointer_default(unique)]
+interface ICoreWebView2Settings6 : ICoreWebView2Settings5 {
+  /// Swiping gesture navigation on touch screen includes:
+  /// 1. Swipe left/right (swipe horizontally) to navigate to previous/next
+  /// page in navigation history.
+  /// 2. Pull to refresh (swipe vertically) the current page. (This feature is
+  /// currently disabled by default in the browser, to enable in WebView2, use
+  /// `put_AdditionalBrowserArguments` api with "--pull-to-refresh" switch).
+  /// The `IsSwipeNavigationEnabled` property enables or disables the ability of the
+  /// end user to use swiping gesture on touch input enabled devices to
+  /// navigate in WebView2. It defaults to `TRUE`.
+  /// When set to `FALSE`, the end user cannot swipe to navigate or pull to refresh.
+  /// This API only affects the overscrolling navigation functionality and has no
+  /// effect on the scrolling interaction used to explore the web content shown
+  /// in WebView2.
+  ///
```

Comment:
Does modification of this setting only apply to the current webview or all webviews in the environment? If just this webview, then no need to specify anything. If it applies to all webviews in the environment then need to say that in the documentation.

Reply:
This setting change only applies to the current webview 😀

Reply:
👍👍